src/current/_data/redirects.yml

```diff
@@ -929,7 +929,7 @@
   sources:
     - import.md
     - cockroach-import.md
-  versions: ['v24.1']
+  versions: ['v24.1', 'v24.2']
```

Comment:
There might be a future tooling requirement here so that you don't have to keep extending this redirect going forward.